p2panda-rs/src/document/document_view.rs

```diff
@@ -1,251 +1,206 @@
-// SPDX-License-Identifier: AGPL-3.0-or-later
+// SPDX-License-Identifier: AGPL-3.view-or-later
```

Comment:
Upsi 😄

Reply:
how the hell did you spot that, that is unreal!!!